contracts/Vesting.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
As I remember IERC20 interface exists in SafeERC20, so its enough to have only SafeERC20